service_event/src/main/scala/no/uio/musit/microservice/event/service/EventOrderingUtils.scala

```diff
@@ -0,0 +1,64 @@
+package no.uio.musit.microservice.event.service
```

Comment:
This file should not be located in the `service` package. It actually belongs in the `domain` package, since it describes behaviour on the domain type `Event`.